docs/video-service-setup.md

```diff
@@ -6,7 +6,8 @@ Midspace uses the Vonage Video API (formerly TokBox OpenTok) for video chat.
    [Vonage Video API](https://www.vonage.co.uk/communications-apis/video/)
    account; then go to `Projects > Create New Project`, choose "Custom", and
    make a note of the API key that is generated.
-1. Generate a secure random value and note it as `VONAGE_WEBHOOK_SECRET`.
+1. If you have not already, generate a secure random value and note it as `VONAGE_WEBHOOK_SECRET`.
+1. Ensure that the Packetriot task (or your preferred tunnel setup) is running locally before proceeding to the next steps. Vonage will not accept the required URLs if they are not exposed via a tunnel.
```

Comment:
The actions service will also need to be running to respond to the test HTTP call from Vonage.

Reply:
I realized that this now adds another layer of complexity as the actions service isn't set up until much later in the process. In one regard, it makes sense to keep the Vonage setup in with the other "Cloud Services" setup steps but logistically, it makes more sense to move this into the "Setting Up Local Working Copy" section after the services have been set up.

As a middle ground, maybe this README could be split into two parts: Part 1 being account creation and Part 2 for after the actions service is running (with a backlink to Part 2 from the actions setup README)?

Reply:
I think your suggestion makes sense here.